en/SUMMARY.md

```diff
@@ -9,6 +9,7 @@
 
 * [Getting the project](getting/project.md)
   * [Using the Installer](getting/installer.md)
+  * [Installing on Windows](getting/windows.md)
```

Comment:
If we plan to have separated instructions for installer based on OS (non-windows and windows), I like to have similar titles

This titles `Installing on Windows` must be similar to the other `Using the Installer on Windows`